Have the problem to make extension

I'm a newbie in linux and have the problem to make the extension on Debian

Could you provide a list of all needed dependencies and full cmake command, please

At now I have installed
- git
- make
- g++
- build-essential postgresql-server-dev-14

but still have some errors. the last :
```cmd
cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Releas PostgreSQL_TYPE_INCLUDE_DIR=./usr/include/postgresql/
CMake Error: The source directory "/pg_ulid/PostgreSQL_TYPE_INCLUDE_DIR=./usr/include/postgresql" does not exist.
Specify --help for usage, or press the help button on the CMake GUI.
```
I don't what to do with it

--- a/Dockerfile.debian
+++ b/Dockerfile.debian
@@ -1,0 +1,22 @@
+FROM debian:bullseye as build
+
+RUN apt-get update && \
+    apt-get install -y \
+      libpq-dev \
+      postgresql-server-dev-all \
+      g++ \
+      cmake \
+      checkinstall
+
+ADD . /pg_ulid
+
+WORKDIR /pg_ulid/build
+
+RUN cmake -DCMAKE_BUILD_TYPE=Release .. && \
+    cmake --build . && \
+    checkinstall -y \
+        --pkgname=pg_ulid \
+        --pkgversion=1.1
+
+FROM scratch AS export-stage
+COPY --from=build /pg_ulid/build/pg-ulid_1.1-1_amd64.deb .

--- a/Readme.md
+++ b/Readme.md
@@ -33,3 +33,8 @@ INSERT INTO foo(value) SELECT generate_series(1, 1000) as value ;
 -- ...
 SELECT ulid_to_string(id), value FROM foo ORDER BY id LIMIT 10;
 ```
+
+### Build in docker
+
+    docker build -f Dockerfile.debian -t deb .
+    sudo find /var/lib/docker/ -name pg-ulid_1.1-1_amd64.deb -type f | xargs sudo ls -hrt | tail -1 | xargs  -i  sudo cp -v  '{}' .

--- a/sql/ulid.control
+++ b/sql/ulid.control
@@ -1,3 +1,3 @@
 comment = 'generator of sequential ULIDs'
-default_version = '1.0'
+default_version = '1.1'
 relocatable = true

--- a/src/ulid.cpp
+++ b/src/ulid.cpp
@@ -3,8 +3,9 @@
 #include <vector>
 
 extern "C" {
-#include <fmgr.h>
 #include <postgres.h>
+
+#include <fmgr.h>
 #include <utils/uuid.h>
 #ifdef PG_MODULE_MAGIC
 PG_MODULE_MAGIC;

